recipes/cuda-cccl/meta.yaml

```
@@ -0,0 +1,94 @@
+{% set name = "cuda-cccl" %}
+{% set version = "12.0.90" %}
+{% set impl_version = "2.0.1" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cccl/{{ platform }}/cuda_cccl-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 37f6e03b6f95579d8e2e96b9ba15eeb1224dcd18b21ad50c3ac249e408152a24  # [linux64]
+  sha256: c6de67acd6a65b24db0383f6330b842ed19b6c27a27a37f828e9f3e96cc96f72  # [ppc64le]
+  sha256: f2a19927c558a9c872fa87eda4c0dc0446e5c4c744df4f846d01f4d4e08c8cd7  # [aarch64]
+  sha256: dd422d4cc8462d112411659a56cc78cb6010d704f83db3e48c55a3f66eed5ec2  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+outputs:
+  - name: cuda-cccl
+    requirements:
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - cuda-cccl_{{ target_platform }} =={{ version }}
+        - cuda-cccl-impl =={{ impl_version }}
+    test:
+      commands:
+        - test -d $PREFIX/targets/{{ target_name }}/lib      # [linux]
+        - test -d $PREFIX/lib/cmake  # [linux]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: 'C++ Core Compute Libraries Documentation and Scripts'
```

CCCL stands for CUDA C++ Core Libraries, not C++ Core Compute Libraries ([source](https://github.com/NVIDIA/cccl)). Also this package doesn't contain documentation or scripts. Please fix on all subpackages/recipes/about sections in this PR.
```suggestion
      summary: 'CUDA C++ Core Libraries'
```

---

So should we just add "CUDA"?

---

Right -- we need the word CUDA somewhere. I'm fine with "CUDA C++ Core Compute Libraries" if that's what the consensus suggests. I also asked the CCCL team to clarify.

---

Confirmed offline with CCCL team: @wmaxey says "CUDA C++ Core Libraries", and I'd go with that for this PR.